fuel/app/migrations/051_add_support_user_role.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Fuel\Migrations;
+
+class Add_support_user_role
+{
+	public function up()
+	{
+		\Materia\Perm_Manager::create_role('support_user');
+	}
+
+	public function down()
```

Comment:
I'll make a new commit to add a delete query here to remove the 'support_user' role from the database.